package.json

```diff
@@ -13,18 +13,42 @@
   "homepage": "https://verdaccio.org",
   "main": "index.js",
   "devDependencies": {
+    "@babel/cli": "7.11.6",
+    "@babel/core": "7.11.6",
+    "@babel/node": "7.10.5",
+    "@babel/plugin-proposal-class-properties": "7.10.4",
+    "@babel/plugin-proposal-decorators": "7.10.5",
+    "@babel/plugin-proposal-export-namespace-from": "7.10.4",
+    "@babel/plugin-proposal-function-sent": "7.10.4",
+    "@babel/plugin-proposal-json-strings": "7.10.4",
     "@babel/plugin-proposal-nullish-coalescing-operator": "7.10.4",
+    "@babel/plugin-proposal-numeric-separator": "7.10.4",
+    "@babel/plugin-proposal-object-rest-spread": "7.11.0",
     "@babel/plugin-proposal-optional-chaining": "7.11.0",
-    "@babel/polyfill": "7.11.5",
+    "@babel/plugin-proposal-throw-expressions": "7.10.4",
```

Comment:
Please are these new dependencies necessary for this PR?

Reply:
Yes, I removed `@verdaccio/babel-preset`, so the babel dependencies need to be on the root from now on. Yarn 2 is strict on this matter.

Reply:
You can still use the preset but it needs to use `require.resolve` for the plugins/presets it adds

Reply:
Thanks, on this specific case, I will use explicit dependencies, much clear for contributors what we are using and easier to update or replace.